classes/lang/KeysReference/MetaLang.php

```diff
@@ -26,7 +26,7 @@
 trans('404 error', 'Shop.Navigation');
 trans('Best sales', 'Shop.Navigation');
 trans('Contact us', 'Shop.Navigation');
-trans('Manufacturers', 'Shop.Navigation');
+trans('Brands', 'Shop.Navigation');
```

Comment:
```suggestion
trans('Brands', 'Shop.Theme.Global');
```

Let's use the existing string.